src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/controller/PartnershipController.java

```diff
@@ -33,7 +35,7 @@
 import static uk.gov.companieshouse.limitedpartnershipsapi.utils.Constants.URL_PARAM_TRANSACTION_ID;
 
 @RestController
-@RequestMapping("/transactions/{" + URL_PARAM_TRANSACTION_ID + "}/limited-partnership/partnership")
+@RequestMapping("/transactions/{" + URL_PARAM_TRANSACTION_ID + "}/limited-partnership")
```

Comment:
i think we should have a separate controller for the submission/limited-partnership as a whole and have the partnershipController just for the /partnership resource

Reply:
Perhaps. What is the difference between a 'limited partnership' and a 'partnership'?

Reply:
they are different parts of the url so could be different resources / sub-resource eg /books/{bookId}/authors/{authorId}

Reply:
The sub resources will be separate resources according to the [LP service design](https://companieshouse.atlassian.net/wiki/spaces/Arch/pages/5055807526/LP+Incorporation+Type+Journeys) so we wouldn't have two is like in the example above. Happy to discuss whether this should be in a separate controller.